src/PrivateRoute/index.jsx

```diff
@@ -22,10 +22,10 @@ const PrivateRoute = ({
     : <Route
       {...rest}
       path={path}
-      render={props => (
+      render={renderProps => (
```

Comment:
I renamed `props` here to avoid confusing eslint - it thought using `props.location.pathname` needed to be declared as prop types for PrivateRoute. Renaming also avoids confusing humans with the same problem.